tests/regression/timetable/analyze_timetable_empty_cell_skip.py

```
@@ -0,0 +1,432 @@
+        ok, buf = cv2.imencode(".png", roi_gray)
+        if ok:
+            tasks.append(buf.tobytes())
+            task_indexes.append(len(cells))
+        else:
+            cell["ocr_status"] = "roi_encode_failed"
+        cells.append(cell)
+
+    if tasks:
+        max_workers = min(ocr_engine.OCR_THREAD_WORKERS, len(tasks)) or 1
+        with ThreadPoolExecutor(max_workers=max_workers, thread_name_prefix="empty_cell_analysis") as executor:
+            for index, lines in zip(task_indexes, executor.map(ocr_engine._ocr_task, tasks)):
```

Ruff (0.15.18)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)